.gitignore

```diff
@@ -130,3 +130,6 @@ dmypy.json
 
 # Model specification files
 src/vivarium_census_prl_synth_pop/model_specifications/*.yaml
+
+# VSCode test cache
+**/.vscode/
```

Comment:
This is odd since I've been using vscode forever and never needed to add this

Reply:
If I use the the vscode debugger it adds this folder and I am lazy and didnt want to remove it every time.

Reply:
I mean, I don't blame you but I haven't needed to do it. weird.